loveMyPet/loveMyPet/Common/Entities/Breed.swift

```diff
@@ -7,14 +7,18 @@
 
 import Foundation
 
-let Breed: Dictionary<Specie, [String]> = [
-    .none: ["Não Escolhido"],
-    .cat: ["Siamês", "Persa", "Bengal", "Maine Coon", "Ragdoll", "SRD"],
-    .dog: ["Labrador Retriever", "Bulldog Francês", "Golden Retriever", "Poodle", "Pastor Alemão", "SRD"],
-    .bird: ["Canário", "Calopsita", "Periquito", "Papagaio", "Agapornis", "SRD"],
-    .rabbit: ["Holland Lop", "Mini Lion", "Angorá", "Fuzzy Lop", "Rex", "SRD"],
-    .hamster: ["Sírio", "Russo", "Roborovski", "Chinês", "Anão", "SRD"],
-    .turtle: ["Tartaruga-de-orelha-vermelha", "Tartaruga-pintada", "Tartaruga-mordedora-comum", "Tartaruga-mordedora-de-musgo", "Tartaruga-de-esporas", "SRD"],
-    .horse: ["Puro Sangue Inglês", "Quarto de Milha", "Andaluz", "Lusitano", "Árabe", "SRD"],
-    .custom: [""]
-]
+struct Breed {
+    
+    static let `for`: Dictionary<Specie, [String]> = [
```

Comment:
Pq desse for? teste trocar por "static let shared: [Specie : String] = [....]"  e talvez fosse interessante dizer que o tipo breed é algo? mas essa ultima parte depende da arquitetura de vocês

Reply:
Breed foi adicionado ao enum de Specie.